### ISSUE TITLE
[defapp] The default terminal subtitle is is difficult to read in High Contrast mode

### ISSUE THREAD

**Issue by jtippet** [microsoft/terminal#10181]
### Windows Terminal version (or Windows build number)

1.9.1391.0

### Other Software

n/a

### Steps to reproduce

Enable High Contrast mode
Open Terminal's Settings page > Startup tab
Try to change the default terminal application

### Expected Behavior

Probably all text should be either solid white or solid black.  It looks like it's currently unconditionally set to SystemBaseMediumColor, which is #99FFFFFF even in HC mode.

### Actual Behavior

Some text is grey, which does not contrast well with the High Contrast accent color:

![image](https://user-images.githubusercontent.com/10259764/119559414-7b196b80-bd57-11eb-8c04-302a29294b82.png)

**Comment by zadjii-msft** [comment]
Great catch, as always!

**Comment by zadjii-msft** [comment]
@miniksa as an FYI

**Comment by miniksa** [comment]
Let me see what I can do here...
